pyproject.toml

```diff
@@ -296,6 +296,7 @@ select = [
   "T10",    # flake8-debugger
   "W",      # pycodestyle
   "YTT",    # flake8-2020
+  "I"       # isort
```

Comment:
I'm sorry for this as this causes a big diff _but_ pylint complains about the import order for all logging imports (since the third-party import of `logging` became a first-party import from `haystack`) and I couldn't bother fixing them manually. In my opinion this should be enabled by default (and we should also run ruff in the CI!)